lib/suspenders/generators/app_generator.rb

```diff
@@ -233,6 +238,14 @@ def remove_routes_comment_lines
       build :remove_routes_comment_lines
     end
 
+    def generate_all
+      run("spring stop")
```

Comment:
This line took me five months to find. I have nothing polite to say about this line.

I'm keeping it as a separate commit when I merge: https://github.com/thoughtbot/suspenders/pull/676/commits/406eedafdf684420f25b15a38e4c64500d584eb4